09a_length/MFW/page2.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Page2</title>
+        <link rel="stylesheet" href = "style.css">
```

Comment:
make sure to not have extra spaces before and after the `=` sign. So instead, do: 
```
<link rel="..." href="style.css">
```